apollo-ios-pagination/Sources/ApolloPagination/GraphQLQueryPager+Convenience.swift

```diff
@@ -1,160 +1,205 @@
 import Apollo
 import ApolloAPI
 
```

Comment:
Can we add a function here for easily converting a `GraphQLPager` to an `AnyGraphQLPager` like a `map` function? Not sure what to call it, maybe `mappingResults`?

So you can do this:
```swift
let pager: AnyGraphQLPager<CustomModel> = makeQueryPager(
    client: client,
    queryProvider: { _ in PageQuery() },
    extractPageInfo: { data in CustomPage(data) }
).mappingResults { result in
 CustomModel(result)
}
```

Reply:
I have those over in `AnyGraphQLPager`, `eraseToAnyPager` -- which allow you to supply a function mapping `Output` to your custom model.